core/src/staked_nodes_updater_service.rs

```diff
@@ -19,21 +20,55 @@ pub struct StakedNodesUpdaterService {
     thread_hdl: JoinHandle<()>,
 }
 
+fn firedancer_write_stake(
+    mvcc: &mut Mvcc,
+    staked_nodes: &HashMap<Pubkey, u64>,
+    staked_nodes_overrides: &HashMap<Pubkey, u64>,
+) {
+    let mut values = staked_nodes.iter().filter(|(pubkey, _)| !staked_nodes_overrides.contains_key(pubkey)).collect::<Vec<_>>();
+    values.extend(staked_nodes_overrides.iter());
+    values.sort_by(|a, b| b.cmp(a));
```

Comment:
why sorted? it still gets deser'd into an `fd_map_dynamic` right?

Reply:
added a comment